package-lock.json

```
@@ -1,6 +1,6 @@
 {
   "name": "arduino-serial-plotter-webapp",
-  "version": "0.0.18",
+  "version": "0.1.0",
```

```suggestion
  "version": "0.0.18",
```

This is wrong (it was a mistake made some time ago when a release was done incorrectly), but it is unrelated to the purpose of this pull request so I think it is better to leave it as is. It will be resolved by the next release of the project.